xml/admin_caasp_object_storage.xml

```diff
@@ -224,7 +224,7 @@ export AWS_SECRET_ACCESS_KEY=$(kubectl -n default get secret ceph-bucket -o yaml
     Run the following commands after you have connected to the Rook toolbox.
    </para>
    <sect3 xml:id="connection-environment-variables">
-    <title>Connection Environment Variables</title>
+    <title>Connecting Environment Variables</title>
```

Comment:
From this title, i'm not sure whether we're connecting variables together, or whether these variables serve for connecting host1 to host2

Reply:
The latter - how would you suggest to rephrase?